install_files/ansible-base/roles/restrict-direct-access/templates/rules_v4

```diff
@@ -60,8 +60,8 @@
 {% endfor -%}
 
 # NTP rules
--A OUTPUT -p udp --sport 123 --dport 123 -m owner --uid-owner root -m state --state NEW,ESTABLISHED,RELATED -j ACCEPT -m comment --comment "ntp"
```

Comment:
While we won't support new installs on  Xenial starting on 1.8.0, it is possible orgs will want to run playbooks against Xenial instances in the time period between 1.8.0 and the time where we eliminate Xenial support (e.g. Onion Service v2 -> v3 migration). The firewall rules are applied on playbook run, but ntp/ntpdate are not explicitly removed as part of the playbook run. Do we understand the implications of having both systemd-timesync and ntp/ntpdate installed at the same time (with presumably ntpdate  being blocked by iptables) ?

Reply:
Yep. I've missed several places where both need to be supported. The CI failure @kushaldas noted is because the `timedatectl` command differs on Xenial, and as you noted I need to split the NTP iptables rules. 

The `systemd-timesyncd` package conflicts with `ntp`, though, so won't be present on Xenial.

Reply:
Actually, `systemd-timesyncd` won't be present on Xenial because that package doesn't exist for Xenial....